### PR TITLE
added wishlist page

### PR DESCRIPTION
**Pull request by omjadhav85**
image

### CODE REVIEW

File: routes/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://subtleui.netlify.app/styles/styles.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/12af55b294.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (nareshpoonia) [inline routes/wishlist.html:14]: You can consider moving font-awesome script to your main css, so you don't have to add it to all of your html files.

File: routes/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      <div class="container">
+        <div class="h3 txt-center">My Wishlist</div>
+        <hr />
+        <div class="wishlist-container">
+          <div class="card card-ecom card-shadow">
+            <div class="card-body badge-container">
+              <div class="card-img">
+                <img
+                  src="../assets/images/sneaker.jpg"
+                  alt=""
```
Comment (nareshpoonia) [inline routes/wishlist.html]: You should add alt-text for accessibility

File: styles/styles.css [hunk 3]
```diff
@@ -2,6 +2,11 @@ body {
...
 }
 
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
 .navbar {
   background-color: var(--pure-white);
   justify-content: space-around;
```
Comment (nareshpoonia) [inline styles/styles.css]: You can consider using utility classes instead, tailwind utility class for the same is justify-around

**Review by nareshpoonia — APPROVED** [review]
Just a few comments, other than that the code looks clean.
Good to merge

File: styles/styles.css [hunk 4]
```diff
@@ -2,6 +2,11 @@ body {
   background-color: #f7f7f7;
 }
 
+.container {
+  max-width: 1000px;
```
Comment (Shivanshsharma01) [inline styles/styles.css:10]: Can use relative units here.

File: styles/styles.css [hunk 5]
```diff
@@ -2,6 +2,11 @@ body {
   background-color: #f7f7f7;
```
Comment (Shivanshsharma01) [inline styles/styles.css]: can use variable here

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean. Can implement suggested changes

File: routes/wishlist.html (hunk 1 above)
Comment (omjadhav85) [reply routes/wishlist.html:14]: Actually i am using version 5.15 of font awesome icons and i dont find any cdn library for it to import in my component library.

File: routes/wishlist.html (hunk 2 above)
Comment (omjadhav85) [reply routes/wishlist.html]: done

File: styles/styles.css (hunk 3 above)
Comment (omjadhav85) [reply styles/styles.css]: will surely add such classes in my CL. Thanks!

File: styles/styles.css (hunk 5 above)
Comment (omjadhav85) [reply styles/styles.css]: done...thanks!

File: styles/styles.css (hunk 4 above)
Comment (omjadhav85) [reply styles/styles.css:10]: Actually i kept it in pixels for better readability

**Pull request merged**